### ISSUE TITLE
Use the Visual Studio Code Color Theme's text selection background colour (set by some themes and users) in the Git Graph View

### ISSUE THREAD

**Issue by niccolomineo** [mhutchie/vscode-git-graph#389]
Hi Michael, I am wondering, do you know what is the JSON key name in a theme for the styling of this particular highlight bg colour in Git Graph? I tried a handful among the default ones, but to no avail.

![Schermata 2020-10-02 alle 17 58 36](https://user-images.githubusercontent.com/5409469/94944776-03ea4b00-04da-11eb-8619-cb392a2ede6e.png)

Cheers,
N

**Comment by mhutchie** [comment]
Hi @niccolomineo,

I've found that the Visual Studio Code colour variable for this is `selection.background`, however the Git Graph View currently doesn't use it. I confirmed this is the correct colour via looking at the text highlighting in the Visual Studio Code Settings Editor.

The reason why the Git Graph View currently doesn't use this variable is because it uses a [Webview](https://code.visualstudio.com/api/extension-guides/webview), and Visual Studio Code provides default styles for what I thought was all low level styles (e.g. link colour, scrollbar colour, body text font & colour etc.) - meaning that extensions only need to style elements that it introduces (e.g. buttons, dialogs etc.). However, for some reason Visual Studio Code isn't setting the default style for text selection (I definitely would have expected this). As the selection background colour is not currently set, I believe it is using Electron's default colour.

It's a simple style change for the Git Graph View to style the highlighted text according to `selection.background`, I'll include it in the next beta release (released sometime this weekend).

FYI: This is the CSS I need to add the the Git Graph View's Stylesheet
```
::selection{
   background-color: var(--vscode-selection-background);
}
```

**Comment by mhutchie** [comment]
This will be available in v1.27.0.

If you'd like to use it before the next release, you can download [v1.27.0-beta.2](https://github.com/mhutchie/vscode-git-graph/releases/tag/v1.27.0-beta.2), and install it following the instructions provided [here](https://github.com/mhutchie/vscode-git-graph/wiki/Downloading-&-Installing-Releases#manual-installation).
